coeus-impl/src/main/java/org/kuali/kra/award/web/struts/action/AwardHomeAction.java

```diff
@@ -237,14 +238,18 @@ public ActionForward save(ActionMapping mapping, ActionForm form, HttpServletReq
             }
             Award oldAward = getAwardVersionService().getActiveAwardVersion(awardDocument.getAward().getAwardNumber());
             AwardAmountInfo aaiNew = awardDocument.getAward().getLastAwardAmountInfo();
-            if (oldAward != null) {
+            if (oldAward != null && oldAward.getLastAwardAmountInfo() != null) {
                 AwardAmountInfo aaiOld = oldAward.getLastAwardAmountInfo();
+                try {
                 aaiNew.setObligatedChange(aaiNew.getAmountObligatedToDate().subtract(aaiOld.getAmountObligatedToDate()));
                 aaiNew.setObligatedChangeDirect(aaiNew.getObligatedTotalDirect().subtract(aaiOld.getObligatedTotalDirect()));
                 aaiNew.setObligatedChangeIndirect(aaiNew.getObligatedTotalIndirect().subtract(aaiOld.getObligatedTotalIndirect()));
                 aaiNew.setAnticipatedChange(aaiNew.getAnticipatedTotalAmount().subtract(aaiOld.getAnticipatedTotalAmount()));
                 aaiNew.setAnticipatedChangeDirect(aaiNew.getAnticipatedTotalDirect().subtract(aaiOld.getAnticipatedTotalDirect()));
                 aaiNew.setAnticipatedChangeIndirect(aaiNew.getAnticipatedTotalIndirect().subtract(aaiOld.getAnticipatedTotalIndirect()));
+                } catch (NullPointerException e) {
+                  aaiNew.resetChangeValues();
+                }
             } else {
```

Comment:
I asked because I think this may have been fixed and out of curiousity. Not saying we should revert now but when we revert, we should test with that jira.